src/tools/clippy/clippy_lints/src/ptr.rs

```diff
@@ -1,19 +1,23 @@
 //! Checks for usage of  `&Vec[_]` and `&String`.
 
-use clippy_utils::diagnostics::{span_lint, span_lint_and_sugg, span_lint_and_then, span_lint_hir_and_then};
+use clippy_utils::diagnostics::{
```

Comment:
This file looks as if it was just re-formatted. Please don't commit formatting changes to Clippy in the Rust repo.